src/map/skill.cpp

```diff
@@ -21984,36 +21978,33 @@ bool skill_produce_mix(map_session_data *sd, uint16 skill_id, t_itemid nameid, i
 					clif_misceffect(&sd->bl,3);
 					break;
 				default: //Those that don't require a skill?
-					if (skill_produce_db[idx].itemlv > 10 && skill_produce_db[idx].itemlv <= 20) { //Cooking items.
+					if (produce->itemlv > 10 && produce->itemlv <= 20) { //Cooking items.
```

Comment:
Should we make the "Cooking Items" a flag in the db? Rather than based on the produce item level

Reply:
I'll link you to #1571. We could address it in this PR or make a follow up once this is converted to yaml. I might be for the latter so it's not hidden in here.